smdebug/pytorch/hook.py

```diff
@@ -155,7 +155,8 @@ def forward_hook(self, module, inputs, outputs):
         # logger.debug("Processing the global step {0} for module {1}".format(self.step, module_name))
 
         # Output input tensor
-        self._write_inputs(module_name, inputs)
+        if isinstance(module, torch.nn.modules.loss._Loss) is False:
```

Comment:
How does it work for functional losses?

Reply:
functional calls record_tensor_value() which writes only outputs